workshops/beat_maker/README.md

```diff
@@ -0,0 +1,418 @@
+
+<div>
+  <div>A</div>
+  <div>B</div>
+  <div>C</div>
+  <div>D</div>
+
+  <div>E</div>
+  <div>F</div>
+  <div>G</div>
+  <div>H</div>
+
+  <div>I</div>
+  <div>J</div>
+  <div>K</div>
+  <div>L</div>
+</div>
+```
+
+![Preview](https://i.ibb.co/72XvNn3/Annotation-2020-09-20-204442.png)
+
+And if we run our code it should look like this.
+Not so compelling right? we will now need to write some CSS which will style our HTML document and make our Beat Maker look prettier, let's do that.
+
+Just before the end of your `<head>` tag, link your CSS file:
+
+```html
+<link rel="stylesheet" href="style.css" />
+```
+
+Your CSS file should now be linked and we can start writing our CSS code.
+
+We're going to start by writing code for our document to change the colors, adding fonts, height, width etc, we are also going to be using a custom font so to do that at the top of our CSS file we will import it using the line `@import 'https://fonts.googleapis.com/css?family=Roboto';`
+
+```css
+@import 'https://fonts.googleapis.com/css?family=Roboto';
+
+html,
+body {
+  background-color: #fff;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+  font-family: 'Roboto', sans-serif;
+}
+```
+
+![Beat Maker Preview](https://i.ibb.co/qgY3jCv/Annotation-2020-09-20-210409.png)
+
+The major thing you will notice when we run our code this time is that our content has ben aligned to the center.
+
+Now what we want to do is style the header to give it spaces and then creates 3 rows and 4 colums to perfectly align the pads and give them a box or container look we will do just that.
+
+```css
+h1 {
+  color: #000;
+  font-size: 5vw;
+  letter-spacing: 6px;
+}
+
+.pad {
+  width: 500px;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.box {
+  width: 100px;
+  height: 100px;
+  margin: 10px 0;
+  box-shadow: 0 8px 6px -6px black;
+  background-color: #444;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 20px;
+  color: rgba(255, 255, 255, 0.4);
+  user-select: none;
+}
+
+.pad-1,
+.pad-2,
+.pad-3,
+.pad-4,
+.pad-5,
+.pad-6,
+.pad-7,
+.pad-8,
+.pad-9,
+.pad-10,
+.pad-11,
+.pad-12 {
+  border: 2px solid;
+}
+```
+
+If you run your code nothing has changed, that because we are using class to group our HTML elements, let's go back to our `index.html` and to our `<div>` we will want to do the following:
+
+```html
+<div class="pad">
+  <div class="box pad-1">A</div>
+  <div class="box pad-2">B</div>
+  <div class="box pad-3">C</div>
+  <div class="box pad-4">D</div>
+
+  <div class="box pad-5">E</div>
+  <div class="box pad-6">F</div>
+  <div class="box pad-7">G</div>
+  <div class="box pad-8">H</div>
+
+  <div class="box pad-9">I</div>
+  <div class="box pad-10">J</div>
+  <div class="box pad-11">K</div>
+  <div class="box pad-12">L</div>
+</div>
+```
+
+![Preview](https://i.ibb.co/TLFcmvk/aasdasd.png)
```

Comment:
Can you use more descriptive alt text throughout? If the images are showing different things, the alt text should be different
```suggestion
![Preview of HTML with layout CSS applied, the pads are in a grid but all gray](https://i.ibb.co/TLFcmvk/aasdasd.png)
```